c/driver/postgresql/copy/writer.h

```diff
@@ -663,15 +766,15 @@ class PostgresCopyStreamWriter {
     return NANOARROW_OK;
   }
 
-  ArrowErrorCode InitFieldWriters(ArrowError* error) {
+  ArrowErrorCode InitFieldWriters(const PostgresConnection* conn, ArrowError* error) {
```

Comment:
nit but should we pass in the whole connection or maybe just `const Resolver&`?

Reply:
That might also simplify the amount of setup you have to do in the unit test?

Reply:
Nice idea